### PR TITLE
Annihilates the floor length braid (and others) with impunity!

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
This PR removes five hairstyles, all quite goofy and ridiculous and physically impossible and straight out of a terrible anime. I would get rid of more, but this is what I am willing to try and get rid of, because I don't think anyone could possibly defend these.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

File: html/changelogs/dansemacabre-flbkiller.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - imagedel: "Annihilates a number of non-sensical and anime-esque hairstyles."
```
Comment (mikomyazaki) [inline html/changelogs/dansemacabre-flbkiller.yml:41]: Might be nice to include a list of the hairstyles you're removing, here.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -581,13 +549,6 @@ Follow by example and make good judgement based on length which list to include
 		gender = MALE
 		chatname = "short hair"
 
-	familyman
```
Comment (KasparoVy) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm:584]: NNNNNOOOOOOOOOOOOO

Comment (alsoandanswer) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm:584]: rip guile reference

Comment (KasparoVy) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm:584]: Max BS

**Pull request merged**